packages/node-integration-tests/suites/sessions/errored-session-aggregate/test.ts

```diff
@@ -12,6 +12,7 @@ test('should aggregate successful, crashed and erroneous sessions', async () =>
   ]);
 
   expect(envelope).toHaveLength(3);
+
```

Comment:
Don't know about this change... Seems risky :D

Reply:
Whoops :D